doc-content/enterprise-only/release-notes/rn-731_fixed-issues-ref.adoc

```diff
@@ -0,0 +1,85 @@
+
+== Process designer
+
+* You cannot open a callable process that contains spaces and extended chars [https://issues.jboss.org/browse/RHPAM-2006[RHPAM-2006]]
+
+
+== Process engine
+
+* {KIE_SERVER} is not removed from the controller view if the server is killed or crashes [https://issues.jboss.org/browse/RHPAM-1654[RHPAM-1654]]
+* The newly added `org.jbpm.correlationkey.length` system property does not truncate the *Value* column in the `CorrelationPropertyInfo` table [https://issues.jboss.org/browse/RHPAM-1961[RHPAM-1961]]
+* The {KIE_SERVER} client does not reuse the marshaller for JMS [https://issues.jboss.org/browse/RHPAM-1988[RHPAM-1988]]
+* An EJB timer error occurs in development mode for an existing process during deploy [https://issues.jboss.org/browse/RHPAM-1985[RHPAM-1985]]
+
+== PMML
+
+* `ScoringStrategiesTest`, `ScorecardProviderTest`, `ExternalObjectModelTest`, `ScoreCardProviderPMMLTest`, and `ScoreCardReasonCodeTest` fail on Windows [https://issues.jboss.org/browse/RHPAM-1815[RHPAM-1815]]
+
+
+endif::[]
+
+ifdef::DM[]
+
+
+== Installer
+
+* When installing only Decision Server, the installer does not create the user specified on the user creation screen [https://issues.jboss.org/browse/RHDM-885[RHDM-885]]
+
+
+== {CENTRAL}
+
+* If you add an action value for a particular guided decision table row, the value is also applied to other rows  [https://issues.jboss.org/browse/RHDM-872[RHDM-872]]
+* Manual changes to the `kmodule.xml` file are overridden [https://issues.jboss.org/browse/RHDM-599[RHDM-599]]
+* In a guilded decision table, an incorrect V&V results when an operator is filled in the row (as opposed to column definition) [https://issues.jboss.org/browse/RHDM-711[RHDM-711]]
+* The {PRODUCT} designer library must be filtered with the RuleFlow filter by default under a SWARM installation [https://issues.jboss.org/browse/RHDM-538[RHDM-538]]
+* The package name white list should contain the projects default package [https://issues.jboss.org/browse/RHPAM-1890[RHPAM-1890]]
+
+== Decision engine
+
+* `MVELConcequenceBuilder` interprets the second appearance of the hash symbol (#) within a string incorrectly as a comment [https://issues.jboss.org/browse/RHDM-856[RHDM-856]]
+* If you use the plus icon (+) to insert a row in a guided rule template, values are missed or changed  [https://issues.jboss.org/browse/RHDM-920[RHDM-920]]
+* {KIE_SERVER} is not removed from the controller view if the server is killed or crashes [https://issues.jboss.org/browse/RHPAM-1654[RHPAM-1654]]
+
+
+
+== OpenShift
+
+* Setting the Maven mirror URL does not exclude {CENTRAL} [https://issues.jboss.org/browse/RHDM-883[RHDM-883]]
+* Kie server timers are not clustered [https://issues.jboss.org/browse/RHPAM-181[RHPAM-181]]
+
+== ScoreCardProviderPMMLTest
+* XStream marshaller does not work with PMML on execution server [https://issues.jboss.org/browse/RHDM-1900[RHDM-1900]]
```

Comment:
I think we should remove this. The referenced JIRA does not exist and from the description I see, this was a test-only change, no impact on users.

Reply:
Removed